### ISSUE TITLE
при каждой выгрузке хранилища в git в изменениях Диаграма Ганта

### ISSUE THREAD

**Issue by Dumsik** [oscript-library/gitsync#59]
Подскажите как побороть чтобы при каждой выгрузке хранилища в git в изменениях не было Диаграммы Ганта.  Ее никто не меняет, а в ней постоянно  меняется реквизит <d3p1:baseData>4294901857</d3p1:baseData>

**Comment by nixel2007** [comment]
Лучший способ - писать в 1с, так это недоработка платформы. 
Еще есть инструмент undiff1c от @pumbaEo, можете попробовать подключить его как прекоммит-хук

**Comment by Dumsik** [comment]
@nixel2007 глянул undiff1c там вроде этот реквизит не обрабатывается. Печальная  ситуация :( напишу в 1с

**Comment by nixel2007** [comment]
> глянул undiff1c там вроде этот реквизит не обрабатывается.

можно же допилить :)

**Comment by Dumsik** [comment]
> можно же допилить :)

Да оно понятно. Просто, там костыль тут костыль ...

**Comment by dmpas** [comment]
> Просто, там костыль тут костыль ...

1С, чё...

**Comment by EvilBeaver** [comment]
А еще было предложение от @dmpas в виде файла .gitsyncignore

**Comment by 1cgh** [comment]
@Dumsik Проблема актуальна еще? Была аналогичная, тоже с диаграммой Ганта.
В 1С писали, итогового ответа не получили.
Но с мая проблема не проявляется. Но пока не выяснили почему)

**Comment by Dumsik** [comment]
@1cgh Да, актуальна. В 1с также написал, скинул примеры. В ответ молчание. :(
